### DISCUSSION TITLE
How to use SWC with nextjs?

### DISCUSSION THREAD

**Discussion by Alii-isk** [swc-project/swc#2097] [Q&A]
how can i setup the swc to work with nextjs project?

**Comment by kdy1** [comment]
If you use canary of next.js, you can enable `experimental.swcLoader` or `experimental.swcMinify`

**Reply by Alii-isk** [reply]
currently i am not using canary  and i know SWC is coming to NeXTjs soon but i really cant not wait 😁

**Comment by smakosh** [comment]
![Screen Shot 2021-09-01 at 14 25 59](https://user-images.githubusercontent.com/20082141/131747424-651ee723-eccc-4e33-a72a-58c0c21330bd.png)
🤔

**Reply by kdy1** [reply]
What's this?

**Reply by smakosh** [reply]
https://github.com/vercel/next.js/issues/28114
